packages/create-sitecore-jss/src/templates/nextjs-multisite/src/lib/sitemap-fetcher/plugins/graphql-sitemap-service.ts

```diff
@@ -0,0 +1,34 @@
+﻿import {
+  GraphQLSitemapService,
+  StaticPath,
+  constants,
+  SiteInfo,
+} from '@sitecore-jss/sitecore-jss-nextjs';
+import config from 'temp/config';
+import { SitemapFetcherPlugin } from '..';
+import { GetStaticPathsContext } from 'next';
+import { siteResolver } from 'lib/site-resolver';
+
+class GraphqlSitemapServicePlugin implements SitemapFetcherPlugin {
+  _graphqlSitemapService: GraphQLSitemapService;
+
+  constructor() {
+    this._graphqlSitemapService = new GraphQLSitemapService({
+      endpoint: config.graphQLEndpoint,
+      apiKey: config.sitecoreApiKey,
+      sites: siteResolver.sites.map((site: SiteInfo) => site.name),
```

Comment:
We should select distinct here (in case the default/config site is the same as a fetched site).

Reply:
I think this should be handled by siteResolver. I can modify siteResolver logic in scope of this PR to have a distinct collection returned in case of Sitecore site and jssAppName are the same?

Reply:
Having duplicate site names is valid tho for the site resolver. This is only a problem here.

Reply:
should have distinct filtering now - please check if this approach is OK